### PR TITLE
docs/hindi: Add Hindi README docs

### PR DESCRIPTION
**Pull request by karthik-script**
## Change Summary

Added Hindi README docs

## Change type

- [X] docs: (changes to the documentation)

## Test/ Verification

Previewed readme.

## Additional information / screenshots (optional)

None

### CODE REVIEW

File: markdown/readme/languages/hindi.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+## PG के साथ डॉकर
+
+```bash
+docker run -d --name noco
+           -v "$(pwd)"/nocodb:/usr/app/data/
+           -p 8080:8080
+            # अपने pg कनेक्शन स्ट्रिंग के साथ बदलें
+           -e NC_DB="pg://host.docker.internal:5432?u=root&p=password&d=d1"
+           # एक यादृच्छिक गुप्त के साथ बदलें
+           -e NC_AUTH_JWT_SECRET="569a1821-0a93-45e8-87ab-eb857f20a010"
```
[the remark this answers is not in the document]
Comment (karthik-script) [reply markdown/readme/languages/hindi.md:55]: example

File: markdown/readme/languages/hindi.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+![8](https://user-images.githubusercontent.com/35857179/194844893-82d5e21b-ae61-41bd-9990-31ad659bf490.png)
+![9](https://user-images.githubusercontent.com/35857179/194844897-cfd79946-e413-4c97-b16d-eb4d7678bb79.png)
+![10](https://user-images.githubusercontent.com/35857179/194844902-c0122570-0dd5-41cf-a26f-6f8d71fefc99.png)
+![11](https://user-images.githubusercontent.com/35857179/194844903-c1e47f40-e782-4f5d-8dce-6449cc70b181.png)
+![12](https://user-images.githubusercontent.com/35857179/194844907-09277d3e-cbbf-465c-9165-6afc4161e279.png)
+
+# विशेषताएँ
+
+### समृद्ध स्प्रेडशीट इंटरफ़ेस
```
[the remark this answers is not in the document]
Comment (karthik-script) [reply markdown/readme/languages/hindi.md:112]: It matches the root readme file

File: markdown/readme/languages/hindi.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+![5](https://user-images.githubusercontent.com/35857179/194844886-a17006e0-979d-493f-83c4-0e72f5a9b716.png)
+![7](https://github.com/nocodb/nocodb/assets/86527202/be64e619-7295-43e2-aa95-cace4462b17f)
+![8](https://github.com/nocodb/nocodb/assets/86527202/4538bf5a-371f-4ec1-a867-8197e5824286)
+
+![8](https://user-images.githubusercontent.com/35857179/194844893-82d5e21b-ae61-41bd-9990-31ad659bf490.png)
+![9](https://user-images.githubusercontent.com/35857179/194844897-cfd79946-e413-4c97-b16d-eb4d7678bb79.png)
+![10](https://user-images.githubusercontent.com/35857179/194844902-c0122570-0dd5-41cf-a26f-6f8d71fefc99.png)
+![11](https://user-images.githubusercontent.com/35857179/194844903-c1e47f40-e782-4f5d-8dce-6449cc70b181.png)
+![12](https://user-images.githubusercontent.com/35857179/194844907-09277d3e-cbbf-465c-9165-6afc4161e279.png)
+
```
[the remark this answers is not in the document]
Comment (karthik-script) [reply markdown/readme/languages/hindi.md:109]: Matches other translated files.

File: markdown/readme/languages/hindi.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+           nocodb/nocodb:latest
+```
+
+## ऑटो-अपस्टाल
+
+ऑटो-अपस्टाल एक एकल कमांड है जो उत्पादन उपयोग के लिए सर्वर पर NocoDB सेट करता है।
+इसके पीछे यह आपके लिए ऑटो-जनरेट डॉकर-कंपोज़ करता है।
+
+```bash
+bash <(curl -sSL http://install.nocodb.com/noco.sh) <(mktemp)
```
[the remark this answers is not in the document]
Comment (karthik-script) [reply markdown/readme/languages/hindi.md:65]: Matches the root Readme file and other translated files.

**Comment by karthik-script** [comment]
Hi @dstala, can you please take a look? Thanks.

**Review by o1lab — APPROVED** [review]

**Pull request merged**